RepoMan/RepoMan/Repository/BitBucketCloudPullRequestReader.cs

```diff
@@ -0,0 +1,359 @@
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Linq;
+using System.Net;
+using System.Net.Http;
+using System.Threading.Tasks;
+using Newtonsoft.Json;
+using RepoMan.Records;
+using ILogger = Serilog.ILogger;
+
+namespace RepoMan.Repository
+{
+    public class BitBucketCloudPullRequestReader :
+        IRepoPullRequestReader
+    {
+        private readonly string _prApiUrl;
+        private readonly string _repoTag;
+        private readonly HttpClient _bbClient;
+        private readonly JsonSerializerSettings _jsonSerializerSettings;
```

Comment:
I set these, but never used them in the `JsonConvert` calls, so I threaded them through instead of ignoring them.